markdown/stencil-docs/developing-further/theme-updates-and-version-control.md

```diff
@@ -1,161 +1,46 @@
-
-I personally find that it is also helpful to migrate *every* theme update into my customized version, as the update is released. If I were to wait, and to allow changes to accumulate and become more substantial, I think it would be more difficult to adequately test – and correctly integrate – the changes. Similarly, I'm grateful to the developer of our base theme (Pixel Union) for updates that have always made small, incremental changes.
-
-### Dynamic Content Injection
-
-_We've also come up with a variety of techniques for dynamically placing content on a page. For example, we can dynamically add tabs to a product page, using content stored on our WebDAV. Similarly, we can inject content from an external file into the body of a page. This allows us to put something that appears in many places, like a sizing chart, in a single place where it can be easily updated._
-
-_Or, we can use the BigCommerce control panel's various HTML editors – for Products, Categories, and Content > Web Pages – to inject content into predefined "dropzones" (custom div's). This allows us to generate a wide variety of content pages using a single custom template._
-
-All of these techniques minimize the complexity of merging base-theme updates into our customized theme. For further details and examples on how we implement these techniques, see [Dynamic Content Rendering on Stencil Storefronts](/stencil-docs/conditional-logic-examples/dynamic-content-rendering).
 
 ---
 
-<a href='#theme-updates_synchronizing-cornerstone' aria-hidden='true' class='block-anchor'  id='theme-updates_synchronizing-cornerstone'><i aria-hidden='true' class='linkify icon'></i></a>
-
-## Synchronizing Your Theme With Cornerstone's Updates
-
-This section addresses two methods that will allow you to keep your customized Stencil theme up to date with Cornerstone's updates, enhancements, and bug fixes:
+<a id="markdown-updates" name="updates"></a>
 
-* Updating Lightly Customized Themes
-* Updating Heavily Customized Themes
+## Theme Updates
 
-We've adapted these tips from one a Stencil theme developer who is an active member of the BigCommerce Community.
+If you are using a Cornerstone theme as your base theme, update notifications are found in our [changelog](). If you are using another theme, please see the theme creator for updates.
 
-### Updating Lightly Customized Themes
+Before updating your theme, always make a backup of the theme. When updating keep a backup of the `config.json`. Config contains all the configurations and customizations for that theme. Never merge an update directly into your theme. Test any new updates before pushing them to a production theme. A good way to do this is keep a separate test theme and compare file changes between them. Then apply the changes to the test theme and check for any problems. 

```

Comment:
Before updating your theme, review the following list items to ensure a successful update: 

- Create a backup of the custom theme. 
- Keep a backup of `config.json`. This file contains all the configurations and many of the customizations for a theme. 
- Never merge an update directly into your custom theme. 
- Test any new updates before pushing the updates to production. A good way to do this is to keep a test theme that mirrors your theme in production and apply any new changes to the test theme. If there are no issues, apply your updates to production.